Gemfile

```diff
@@ -4,6 +4,8 @@ ENV['grape'] ||= '0.16.2'
 ENV['rails'] ||= '5.0.0'
 ENV['doorkeeper'] ||= '4.0.0'
 
+ruby '>=2.2.2' if ENV['rails'][0].to_i > 4
```

Comment:
Not sure if this should be in the Gemfile. Rails will not run in ruby < 2.2.2 anyways.